### ISSUE TITLE
KeyError: 'twoColumnSearchResultsRenderer' when using search method [BUG] 

### ISSUE THREAD

**Issue by ghost** [pytube/pytube#1106]
**Describe the bug**
I'm facing some errors when I'm trying to use search method

**To Reproduce**
run the following code
```python
s = Search('YouTube Rewind')
print(s.results)
```

**Expected behavior**
get the search results

**Output**
```python
Traceback (most recent call last):
  File "C:\Users\UserName\PycharmProjects\myProject\venv\lib\site-packages\pytube\contrib\search.py", line 94, in fetch_and_parse
    sections = raw_results['contents']['twoColumnSearchResultsRenderer'][
KeyError: 'twoColumnSearchResultsRenderer'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:/Users/UserName/PycharmProjects/myProject/main.py", line 18, in <module>
    main()
  File "C:/Users/UserName/PycharmProjects/myProject/main.py", line 14, in main
    YouTubeMainPageSearch()
  File "C:/Users/UserName/PycharmProjects/myProject/main.py", line 6, in __init__
    self.main()
  File "C:/Users/UserName/PycharmProjects/myProject/main.py", line 10, in main
    print(s.results)
  File "C:\Users\UserName\PycharmProjects\myProject\venv\lib\site-packages\pytube\contrib\search.py", line 62, in results
    videos, continuation = self.fetch_and_parse()
  File "C:\Users\UserName\PycharmProjects\myProject\venv\lib\site-packages\pytube\contrib\search.py", line 97, in fetch_and_parse
    sections = raw_results['onResponseReceivedCommands'][0][
KeyError: 'onResponseReceivedCommands'

```

**System information**

- Python version (Python 3.7.5)
- Pytube version (11.0.0)
- Command used to install pytube :`pip install pytube`

**Comment by priamoryki** [comment]
It's #1082.

**Comment by tfdahlin** [comment]
This should be resolved in v11.0.1, please update with `python3 -m pip install --upgrade pytube`.
